### ISSUE TITLE
How does channel table work?

### ISSUE THREAD

**Issue by marco73** [mattdibi/redox-w-firmware#24]
I like the Redox wireless so much, I made a second one. What I already thought, all keyboards are connected to each other as they share the same channels. I've looked into the answer you gave on another topic but cannot find out how to interpret the channel_table.

As is in code: 
`#ifdef COMPILE_LEFT`
`static uint8_t channel_table[3]={4, 42, 77};`
`#endif`
`#ifdef COMPILE_RIGHT`
`static uint8_t channel_table[3]={25, 63, 33};`
`#endif`

According to NRF documentation there are 31 channels,. So I assume the first entry is the channel and the second value is 38 higher (bandwidth?) but for the other value in the channel table I cannot find anything on how to interpret/configure it. 

Can someone help me with this or have a better method to connect a receiver and transmitters to each other?

**Comment by mattdibi** [comment]
Hi,

```
#ifdef COMPILE_LEFT
static uint8_t channel_table[3]={4, 42, 77};
#endif
```

are the channels used by the left transmitter.

```
#ifdef COMPILE_RIGHT
static uint8_t channel_table[3]={25, 63, 33};
#endif
```

are the channels used by the right transmitter.
The receiver will use all of them as you can see [here](https://github.com/mattdibi/redox-w-firmware/blob/a193645e1b7f70f1637878c5ae9830aebc2de828/redox-w-receiver-basic/main.c#L47).
You'll need to change them consistently for receiver and transmitters.

You might also want to change the address of the keyboard. References:
- [Receiver](https://github.com/mattdibi/redox-w-firmware/blob/a193645e1b7f70f1637878c5ae9830aebc2de828/redox-w-receiver-basic/main.c#L91)
- [Transmitters](https://github.com/mattdibi/redox-w-firmware/blob/a193645e1b7f70f1637878c5ae9830aebc2de828/redox-w-keyboard-basic/main.c#L213)

**Comment by marco73** [comment]
I see I read the channels_table wrong. Found the correct NRF51 documentation. The NRF51 has 80 channels and the table hold just three channels which it will use. So I changed the channels to different channels but got a lot of weird/unwanted behavior on other channels.

In the end I changed the base_address_0 and base_address_1 and kept the channels_tables unchanged. Both keyboards don't interfere with each other and I did not notice any lag when both keyboards were used.

Thanks for the help and for sharing the designs!
